### PR TITLE
Fix docker compose

### PR DESCRIPTION
**Pull request by benjefferies**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
 
 after_success:
   - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker build -t echosoft/taxi .
-    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-    docker push echosoft/taxi:latest
-    fi
\ No newline at end of file
+    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
```
Comment (DaJaffaMan) [inline .travis.yml:11]: where are these being set? do we have a private wiki page or something to get setup?

Comment (DaJaffaMan) [inline .travis.yml:11]: Is this what fixed the build issue?

Comment (benjefferies) [reply .travis.yml:11]: In travis environmental variables

Comment (benjefferies) [reply .travis.yml:11]: The fix for travis was mainly the semi colons

File: src/main/resources/defaults.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-elasticsearchConfig.host=localhost
-elasticsearchConfig.port=9200
\ No newline at end of file
+elasticsearch.host=elasticsearch
```
Comment (benjefferies) [inline src/main/resources/defaults.properties:1]: Changing the config structure was the main fix for docker compose

**Pull request merged**